PlantFriends.ino

```diff
@@ -73,11 +79,13 @@ typedef struct {
   char mqtt_username[33]; // Just set these to 32 as well
   char mqtt_password[33];
   char mqtt_base_topic[33];
+  uint32_t cal_dry;
+  uint32_t cal_wet;
 
   // Can be set via MQTT and the web interface
   uint8_t sleep_time;
   uint16_t watering_delay;
-  uint16_t watering_threshold;
+  float watering_threshold_pct;
```

Comment:
I vote for adding a CRC value while we are at it. I can add to this PR before we make a new release.

Reply:
Added as a task in the todo